src/core/Akka.Streams.Tests/Dsl/FramingSpec.cs

```diff
@@ -384,6 +440,39 @@ public void Length_field_based_framing_must_fail_the_stage_on_negative_length_fi
                 .Should().Throw<Framing.FramingException>()
                 .WithMessage("Decoded frame header reported negative size -4");
         }
+        
+        [Fact]
+        public void Length_field_based_framing_must_ignore_length_field_value_when_provided_computeFrameSize()
+        {
+            int ComputeFrameSize(IReadOnlyList<byte> offset, int length) => 8;
+
+            var tempArray = new byte[4].PutInt(unchecked((int)0xFF010203), order: ByteOrder.LittleEndian);
+            Array.Resize(ref tempArray, 8);
+            var bs = ByteString.FromBytes(tempArray.PutInt(checked(0x04050607), order: ByteOrder.LittleEndian));
```

Comment:
We're missing the [ByteStringBuilder](https://github.com/akka/akka/blob/d703a2afe0e0c6dc8f1c0321b0f4652600e2ae99/akka-actor/src/main/scala-2.12/akka/util/ByteString.scala#L1010) which would've made this chunk of code more readable

Reply:
I believe that was removed in 1.3 or 1.4 and I don't recall why